### PR TITLE
Swagger for REST w/o `dropwizard-swagger`

### PR DESCRIPTION
**Pull request by mpenick**

### CODE REVIEW

**Comment by mpenick** [comment]
Never mind, It could be a pretty big undertaking to switch to 3.0. I'm not sure it's that beneficial right now.

File: restapi/pom.xml [hunk 1]
```diff
@@ -172,6 +138,75 @@
...
+            <id>copy-resources</id>
+            <phase>prepare-package</phase>
+            <goals>
+              <goal>copy-resources</goal>
+            </goals>
+            <configuration>
+              <outputDirectory>${project.build.directory}/classes/swagger-ui</outputDirectory>
+              <resources>
+                <resource>
+                  <directory>${project.build.directory}/swagger-ui/META-INF/resources/webjars/swagger-ui/${swagger-ui.version}</directory>
```
Comment (dougwettlaufer) [inline restapi/pom.xml:202]: What does this mean for when we need to pass in tokens and display models based on db content?

File: restapi/src/main/java/io/stargate/web/impl/Server.java [hunk 2]
```diff
@@ -104,7 +116,21 @@ protected void configure() {
     environment.jersey().register(TablesResource.class);
     environment.jersey().register(KeyspacesResource.class);
     environment.jersey().register(ColumnsResource.class);
-    environment.jersey().register(SwaggerStaticResource.class);
+
+    environment.jersey().register(ApiListingResource.class);
```
Comment (dougwettlaufer) [inline restapi/src/main/java/io/stargate/web/impl/Server.java:120]: Does this still return raw json/yaml with `/swagger.json` and `/swagger.yaml`?

File: restapi/pom.xml (hunk 1 above)
Comment (mpenick) [reply restapi/pom.xml:202]: Not sure what you mean here.

File: restapi/pom.xml [hunk 3]
```diff
@@ -172,6 +138,75 @@
...
+            </goals>
+          </execution>
+        </executions>
+        <configuration>
+          <file>${project.build.directory}/swagger-ui/META-INF/resources/webjars/swagger-ui/${swagger-ui.version}/index.html</file>
+          <replacements>
+            <replacement>
+              <token>https://petstore.swagger.io/v2/swagger.json</token>
+              <value>/swagger.json</value>
+            </replacement>
```
Comment (dougwettlaufer) [inline restapi/pom.xml:183]: We'll need to be careful about this when we upgrade versions.

File: restapi/src/main/java/io/stargate/web/impl/Server.java (hunk 2 above)
Comment (mpenick) [reply restapi/src/main/java/io/stargate/web/impl/Server.java:120]:
```

Comment (dougwettlaufer) [reply restapi/src/main/java/io/stargate/web/impl/Server.java:120]: Sweet

**Review by dougwettlaufer — APPROVED** [review]

**Pull request merged**